src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,106 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static StorageDao storageDao;
+    private static User defaultUser;
+    private static String correctLogin;
+    private static String checkLogin;
+    private static int correctAge;
+    private static int youngAge;
+    private static int incorrectAge;
+    private static String correctPassword;
+    private static String wrongPassword;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void beforeEach() {
```

Comment:
you can use BeforeEach for the INITIALIZATION DEFAULT USER (not constant fields)
<img width="724" alt="image" src="https://user-images.githubusercontent.com/96740815/198040570-50a08544-3a2e-4564-9e4e-a37798f36f87.png">